packages/client/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import * as RadixDialog from '@radix-ui/react-dialog'
+
+interface DialogProps {
+  open: boolean
+  onClose: () => void
+  children: React.ReactNode
+}
+
+const Dialog = (props: DialogProps) => {
+  const {open, onClose, children} = props
```

Comment:
+1 in case we'd want to override some style, position, whatever, I'd expose `className` here. as overriding tailwind classnames isn't too easy in case there's already a related class used, I'd suggest using [dcastil/tailwind-merge: Merge Tailwind CSS classes without style conflicts](https://github.com/dcastil/tailwind-merge)